ext/session/session.c

```diff
@@ -2103,7 +2103,7 @@ PHP_FUNCTION(session_save_path)
 	zend_string *name = NULL;
 	zend_string *ini_name;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|S", &name) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|S!", &name) == FAILURE) {
```

Comment:
It seems to me we should expect `|P!` here, which allows us to drop the check below.

Reply:
Good idea! I've just fixed it